VERSIONING.md

```diff
@@ -33,3 +33,35 @@ Compose version | Kotlin version
 1.0.0 | 1.5.31
 1.1.1 | 1.6.20
 1.2.0 | 1.7.10 - 1.7.20 (Desktop, Android), 1.7.10 (JS)
+
+### Relation to Jetpack Compose and Compose Multiplatform release cycle
+
+Compose Multiplatform shares a lot of code with [Jetpack Compose](https://developer.android.com/jetpack/compose) for Android developed by Google.
+We keep our release cycles aligned, so the common part is properly tested and stabilized.
+
+When the new minor version of Jetpack Compose is released, we pick the release commit, use it as a base for [Compose Multiplatform fork](https://github.com/JetBrains/androidx), finish new platform features, stabilize all platforms, and release Compose Multiplatform after some time.
+The gap between the Compose Multiplatform release and the Jetpack Compose release is usually 1-3 months.
+
+When you build your application for Android, the artifacts published by Google are used.
+
+In the next table you can see versions of the corresponding Jetpack Compose libraries.
+These are the versions on which we based Compose Multiplatform, and the versions used when you build your application for Android.
+
+Compose Multiplatform | Jetpack Compose Compiler | Jetpack Compose Runtime, UI, Foundation, Material | Jetpack Compose Material 3
+--- |--------------------|---------------------------------------------------| ---
+1.0.0 | 1.1.0-rc03 | 1.1.0-rc03                                        | 1.0.0-alpha03
+1.1.1 | 1.1.0 | 1.1.0                                             | 1.0.0-alpha05
+1.2.0 | 1.3.2 | 1.2.1                                             | 1.0.0-alpha14
```

Comment:
Do we plan to add new rows to this table for every release? 

I guess the answer is yes. Would be good to not forget about it. Maybe some checklist for a release could help.

Reply:
Unclear why we are maintaining this table.  What is a user supposed to do with this table?

Reply:
> Do we plan to add new rows to this table for every release?

Yes, we should update this table for every stable release.

> Maybe some checklist for a release could help.

Good idea, I will make such checklist.

> What is a user supposed to do with this table?

Knowing version of Jetpack Compose is useful in some cases:
- to know the current issues, and when they are/will be fixed
- to watch Jetpack Compose changelog
- to investigate compatibility issues (for example, if users use some Android library that depend on the other version of Jetpack Compose)

There were a few questions in Slack/GitHub about on which version we were based.

Reply:
All of those sound like things that are almost release-notes, and most relevant when the user is upgrading.  Eg. "know the current issues and when they are/will be fixed", "watch jetpack compose changelog", "investigate compatibility issues".  I wonder if instead of a hard-to-discover table in the middle of a doc on release cycles, maybe the release notes for each release could contain a mention of which version of Compose for Android is being referenced.

Reply:
> maybe the release notes for each release could contain a mention of which version of Compose for Android is being referenced.

Good idea! I will then prepare a change in our `CHANGELOG.md` adding something like this for each version:

### Updated dependencies
This version of Compose Multiplatform is based on:
* Jetpack Compose 1.2.1
* Jetpack Compose Compiler 1.3.2
* Jetpack Compose Material3 1.0.0-alpha14

Reply:
Moved to [CHANGELOG](https://github.com/JetBrains/compose-jb/pull/2374/files)